### PR TITLE
chore(AWS): match all AWS resource types with SecurityHub supported types in metadata

### PR DESCRIPTION
**Pull request by puchy22**
### Context

The ResourceType from metadata did not match with [SecurityHub reference](https://docs.aws.amazon.com/securityhub/latest/userguide/asff-resources.html)

### Description

Change very AWS Check metadata ResourceType.

### Checklist

- Are there new checks included in this PR? No
    - If so, do we need to update permissions for the provider? No.
- [x] Review if the code is being covered by tests.
- [x] Review if code is being documented following this specification https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jfagoagas — COMMENTED** [review]
Is this still compatible with AWS Security Hub? Please review this guide https://docs.aws.amazon.com/securityhub/latest/userguide/asff-resources.html

Probably we will need to add a validator/transform for the ASFF format.

I'm thinking if we should have a list of the AWS Security Hub valid types.

File: docs/tutorials/reporting.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ The following code is an example output of the [JSON-ASFF](https://docs.aws.amaz
     "Description": "Ensure RDS instances have minor version upgrade enabled.",
     "Resources": [
         {
-            "Type": "AwsRdsDbInstance",
+            "Type": "AwsRdsDbInstanceDetails",
```
Comment (MrCloudSec) [inline docs/tutorials/reporting.md]:
```suggestion
            "Type": "AwsRdsDbInstance",
```

File: prowler/providers/aws/services/acm/acm_certificates_expiration_check/acm_certificates_expiration_check.metadata.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:acm:region:account-id:certificate/resource-id",
   "Severity": "high",
-  "ResourceType": "AwsCertificateManagerCertificate",
+  "ResourceType": "AwsCertificateManagerCertificateDetails",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/acm/acm_certificates_expiration_check/acm_certificates_expiration_check.metadata.json]:
```suggestion
  "ResourceType": "AwsCertificateManagerCertificate",
```

File: prowler/providers/aws/services/acm/acm_certificates_transparency_logs_enabled/acm_certificates_transparency_logs_enabled.metadata.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:acm:region:account-id:certificate/resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsCertificateManagerCertificate",
+  "ResourceType": "AwsCertificateManagerCertificateDetails",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/acm/acm_certificates_transparency_logs_enabled/acm_certificates_transparency_logs_enabled.metadata.json]:
```suggestion
  "ResourceType": "AwsCertificateManagerCertificate",
```

File: prowler/providers/aws/services/athena/athena_workgroup_encryption/athena_workgroup_encryption.metadata.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:athena:region:account-id:workgroup/resource-id",
   "Severity": "medium",
-  "ResourceType": "WorkGroup",
+  "ResourceType": "AwsAthenaWorkGroupConfigurationResultConfigurationEncryptionConfiguration",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/athena/athena_workgroup_encryption/athena_workgroup_encryption.metadata.json]:
```suggestion
  "ResourceType": "AwsAthenaWorkGroup",
```

File: prowler/providers/aws/services/athena/athena_workgroup_enforce_configuration/athena_workgroup_enforce_configuration.metadata.json [hunk 5]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:athena:region:account-id:workgroup/resource-id",
   "Severity": "medium",
-  "ResourceType": "WorkGroup",
+  "ResourceType": "AwsAthenaWorkGroupConfiguration",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/athena/athena_workgroup_enforce_configuration/athena_workgroup_enforce_configuration.metadata.json]:
```suggestion
  "ResourceType": "AwsAthenaWorkGroup",
```

File: prowler/providers/aws/services/awslambda/awslambda_function_inside_vpc/awslambda_function_inside_vpc.metadata.json [hunk 6]
```diff
@@ -7,7 +7,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:lambda:region:account-id:function/function-name",
   "Severity": "low",
-  "ResourceType": "AwsLambdaFunction",
+  "ResourceType": "AwsLambdaFunctionVpcConfig",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/awslambda/awslambda_function_inside_vpc/awslambda_function_inside_vpc.metadata.json]:
```suggestion
  "ResourceType": "AwsLambdaFunction",
```

File: prowler/providers/aws/services/awslambda/awslambda_function_no_secrets_in_code/awslambda_function_no_secrets_in_code.metadata.json [hunk 7]
```diff
@@ -7,7 +7,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:lambda:region:account-id:function/function-name",
   "Severity": "critical",
-  "ResourceType": "AwsLambdaFunction",
+  "ResourceType": "AwsLambdaFunctionCode",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/awslambda/awslambda_function_no_secrets_in_code/awslambda_function_no_secrets_in_code.metadata.json]:
```suggestion
  "ResourceType": "AwsLambdaFunction",
```

File: prowler/providers/aws/services/cloudformation/cloudformation_stack_outputs_find_secrets/cloudformation_stack_outputs_find_secrets.metadata.json [hunk 8]
```diff
@@ -7,7 +7,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:cloudformation:region:account-id:stack/resource-id",
   "Severity": "critical",
-  "ResourceType": "AwsCloudFormationStack",
+  "ResourceType": "AwsCloudFormationStackOutputs",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cloudformation/cloudformation_stack_outputs_find_secrets/cloudformation_stack_outputs_find_secrets.metadata.json]:
```suggestion
  "ResourceType": "AwsCloudFormationStack",
```

File: prowler/providers/aws/services/cloudwatch/cloudwatch_changes_to_network_acls_alarm_configured/cloudwatch_changes_to_network_acls_alarm_configured.metadata.json [hunk 9]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:cloudwatch:region:account-id:certificate/resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsCloudTrailTrail",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cloudwatch/cloudwatch_changes_to_network_acls_alarm_configured/cloudwatch_changes_to_network_acls_alarm_configured.metadata.json]:
```suggestion
  "ResourceType": "AwsCloudWatchAlarm",
```

File: prowler/providers/aws/services/cloudwatch/cloudwatch_changes_to_network_gateways_alarm_configured/cloudwatch_changes_to_network_gateways_alarm_configured.metadata.json [hunk 10]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:cloudwatch:region:account-id:certificate/resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsCloudTrailTrail",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cloudwatch/cloudwatch_changes_to_network_gateways_alarm_configured/cloudwatch_changes_to_network_gateways_alarm_configured.metadata.json]:
```suggestion
  "ResourceType": "AwsCloudWatchAlarm",
```

File: prowler/providers/aws/services/cloudwatch/cloudwatch_changes_to_network_route_tables_alarm_configured/cloudwatch_changes_to_network_route_tables_alarm_configured.metadata.json [hunk 11]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:cloudwatch:region:account-id:certificate/resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsCloudTrailTrail",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cloudwatch/cloudwatch_changes_to_network_route_tables_alarm_configured/cloudwatch_changes_to_network_route_tables_alarm_configured.metadata.json]:
```suggestion
  "ResourceType": "AwsCloudWatchAlarm",
```

File: prowler/providers/aws/services/cloudwatch/cloudwatch_changes_to_vpcs_alarm_configured/cloudwatch_changes_to_vpcs_alarm_configured.metadata.json [hunk 12]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:cloudwatch:region:account-id:certificate/resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsCloudTrailTrail",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cloudwatch/cloudwatch_changes_to_vpcs_alarm_configured/cloudwatch_changes_to_vpcs_alarm_configured.metadata.json]:
```suggestion
  "ResourceType": "AwsCloudWatchAlarm",
```

File: prowler/providers/aws/services/cloudwatch/cloudwatch_cross_account_sharing_disabled/cloudwatch_cross_account_sharing_disabled.metadata.json [hunk 13]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:cloudwatch:region:account-id:certificate/resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsCloudWatch",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cloudwatch/cloudwatch_cross_account_sharing_disabled/cloudwatch_cross_account_sharing_disabled.metadata.json]:
```suggestion
  "ResourceType": "AwsAccount",
```

File: tests/lib/check/fixtures/bulk_checks_metadata.py [hunk 14]
```diff
@@ -41,7 +41,7 @@
         SubServiceName="subnet",
         ResourceIdTemplate="arn:partition:service:region:account-id:resource-id",
         Severity="medium",
-        ResourceType="AwsEc2Vpc",
+        ResourceType="AwsEc2VpcDetails",
```
Comment (MrCloudSec) [inline tests/lib/check/fixtures/bulk_checks_metadata.py]:
```suggestion
        ResourceType="AwsEc2Vpc",
```

File: tests/lib/check/fixtures/bulk_checks_metadata.py [hunk 15]
```diff
@@ -69,7 +69,7 @@
...
```
Comment (MrCloudSec) [inline tests/lib/check/fixtures/bulk_checks_metadata.py]:
```suggestion
        ResourceType="AwsEc2Vpc",
```

File: prowler/providers/aws/services/codebuild/codebuild_project_no_secrets_in_variables/codebuild_project_no_secrets_in_variables.metadata.json [hunk 16]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
   "Severity": "critical",
-  "ResourceType": "AwsCodeBuildProject",
+  "ResourceType": "AwsCodeBuildProjectEnvironmentEnvironmentVariables",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/codebuild/codebuild_project_no_secrets_in_variables/codebuild_project_no_secrets_in_variables.metadata.json]:
```suggestion
  "ResourceType": "AwsCodeBuildProject",
```

File: prowler/providers/aws/services/codebuild/codebuild_project_source_repo_url_no_sensitive_credentials/codebuild_project_source_repo_url_no_sensitive_credentials.metadata.json [hunk 17]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
   "Severity": "critical",
-  "ResourceType": "AwsCodeBuildProject",
+  "ResourceType": "AwsCodeBuildProjectSource",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/codebuild/codebuild_project_source_repo_url_no_sensitive_credentials/codebuild_project_source_repo_url_no_sensitive_credentials.metadata.json]:
```suggestion
  "ResourceType": "AwsCodeBuildProject",
```

File: prowler/providers/aws/services/cognito/cognito_user_pool_advanced_security_enabled/cognito_user_pool_advanced_security_enabled.metadata.json [hunk 18]
```diff
@@ -7,7 +7,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:aws:cognito-idp:region:account:userpool/userpool-id",
   "Severity": "medium",
-  "ResourceType": "AwsCognitoUserPool",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cognito/cognito_user_pool_advanced_security_enabled/cognito_user_pool_advanced_security_enabled.metadata.json]:
```suggestion
  "ResourceType": "AwsCognitoUserPool",
```

File: prowler/providers/aws/services/cognito/cognito_user_pool_blocks_compromised_credentials_sign_in_attempts/cognito_user_pool_blocks_compromised_credentials_sign_in_attempts.metadata.json [hunk 19]
```diff
@@ -7,7 +7,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:aws:cognito-idp:region:account:userpool/userpool-id",
   "Severity": "medium",
-  "ResourceType": "AwsCognitoUserPool",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cognito/cognito_user_pool_blocks_compromised_credentials_sign_in_attempts/cognito_user_pool_blocks_compromised_credentials_sign_in_attempts.metadata.json]:
```suggestion
  "ResourceType": "AwsCognitoUserPool",
```

File: prowler/providers/aws/services/cognito/cognito_user_pool_blocks_potential_malicious_sign_in_attempts/cognito_user_pool_blocks_potential_malicious_sign_in_attempts.metadata.json [hunk 20]
```diff
@@ -7,7 +7,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:aws:cognito-idp:region:account:userpool/userpool-id",
   "Severity": "medium",
-  "ResourceType": "AwsCognitoUserPool",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cognito/cognito_user_pool_blocks_potential_malicious_sign_in_attempts/cognito_user_pool_blocks_potential_malicious_sign_in_attempts.metadata.json]:
```suggestion
  "ResourceType": "AwsCognitoUserPool",
```

File: prowler/providers/aws/services/cognito/cognito_user_pool_client_prevent_user_existence_errors/cognito_user_pool_client_prevent_user_existence_errors.metadata.json [hunk 21]
```diff
@@ -7,7 +7,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:aws:cognito-idp:region:account:userpool/userpool-id",
   "Severity": "medium",
-  "ResourceType": "AwsCognitoUserPoolClient",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cognito/cognito_user_pool_client_prevent_user_existence_errors/cognito_user_pool_client_prevent_user_existence_errors.metadata.json]:
```suggestion
  "ResourceType": "AwsCognitoUserPool",
```

File: prowler/providers/aws/services/cognito/cognito_user_pool_client_token_revocation_enabled/cognito_user_pool_client_token_revocation_enabled.metadata.json [hunk 22]
```diff
@@ -7,7 +7,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:aws:cognito-idp:region:account:userpool/userpool-id",
   "Severity": "medium",
-  "ResourceType": "AwsCognitoUserPoolClient",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cognito/cognito_user_pool_client_token_revocation_enabled/cognito_user_pool_client_token_revocation_enabled.metadata.json]:
```suggestion
  "ResourceType": "AwsCognitoUserPool",
```

File: prowler/providers/aws/services/cognito/cognito_user_pool_deletion_protection_enabled/cognito_user_pool_deletion_protection_enabled.metadata.json [hunk 23]
```diff
@@ -7,7 +7,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:aws:cognito-idp:region:account:userpool/userpool-id",
   "Severity": "medium",
-  "ResourceType": "AwsCognitoUserPool",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/cognito/cognito_user_pool_deletion_protection_enabled/cognito_user_pool_deletion_protection_enabled.metadata.json]:
```suggestion
  "ResourceType": "AwsCognitoUserPool",
```

File: prowler/providers/aws/services/ec2/ec2_instance_imdsv2_enabled/ec2_instance_imdsv2_enabled.metadata.json [hunk 24]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsEc2Instance",
+  "ResourceType": "AwsEc2InstanceMetadataOptions",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/ec2/ec2_instance_imdsv2_enabled/ec2_instance_imdsv2_enabled.metadata.json]:
```suggestion
  "ResourceType": "AwsEc2Instance",
```

File: prowler/providers/aws/services/ec2/ec2_instance_port_memcached_exposed_to_internet/ec2_instance_port_memcached_exposed_to_internet.metadata.json [hunk 25]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "instance",
   "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
   "Severity": "critical",
-  "ResourceType": "AwsEc2Instance",
+  "ResourceType": "AwsEc2SecurityGroupIpPermission",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/ec2/ec2_instance_port_memcached_exposed_to_internet/ec2_instance_port_memcached_exposed_to_internet.metadata.json]:
```suggestion
  "ResourceType": "AwsEc2SecurityGroup",
```

File: prowler/providers/aws/services/ec2/ec2_networkacl_allow_ingress_tcp_port_22/ec2_networkacl_allow_ingress_tcp_port_22.metadata.json [hunk 26]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "networkacl",
   "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsEc2NetworkAcl",
+  "ResourceType": "AwsEc2NetworkAclEntry",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/ec2/ec2_networkacl_allow_ingress_tcp_port_22/ec2_networkacl_allow_ingress_tcp_port_22.metadata.json]:
```suggestion
  "ResourceType": "AwsEc2NetworkAcl",
```

File: prowler/providers/aws/services/ec2/ec2_networkacl_allow_ingress_tcp_port_3389/ec2_networkacl_allow_ingress_tcp_port_3389.metadata.json [hunk 27]
```diff
@@ -9,7 +9,7 @@
   "SubServiceName": "networkacl",
   "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsEc2NetworkAcl",
+  "ResourceType": "AwsEc2NetworkAclEntry",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/ec2/ec2_networkacl_allow_ingress_tcp_port_3389/ec2_networkacl_allow_ingress_tcp_port_3389.metadata.json]:
```suggestion
  "ResourceType": "AwsEc2NetworkAcl",
```

File: prowler/providers/aws/services/ecr/ecr_registry_scan_images_on_push_enabled/ecr_registry_scan_images_on_push_enabled.metadata.json [hunk 28]
```diff
@@ -10,7 +10,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:ecr:region:account-id:resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsEcrRegistry",
+  "ResourceType": "AwsEcrRepositoryImageScanningConfiguration",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/ecr/ecr_registry_scan_images_on_push_enabled/ecr_registry_scan_images_on_push_enabled.metadata.json]:
```suggestion
  "ResourceType": "Other",
```

File: prowler/providers/aws/services/efs/efs_encryption_at_rest_enabled/efs_encryption_at_rest_enabled.metadata.json [hunk 29]
```diff
@@ -11,7 +11,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsEFSFileSystem",
+  "ResourceType": "AwsEfsAccessPoint",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/efs/efs_encryption_at_rest_enabled/efs_encryption_at_rest_enabled.metadata.json]:
```suggestion
  "ResourceType": "AwsEfsFileSystem",
```

File: prowler/providers/aws/services/efs/efs_have_backup_enabled/efs_have_backup_enabled.metadata.json [hunk 30]
```diff
@@ -11,7 +11,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsEFSFileSystem",
+  "ResourceType": "AwsEfsAccessPoint",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/efs/efs_have_backup_enabled/efs_have_backup_enabled.metadata.json]:
```suggestion
  "ResourceType": "AwsEfsFileSystem",
```

File: prowler/providers/aws/services/efs/efs_not_publicly_accessible/efs_not_publicly_accessible.metadata.json [hunk 31]
```diff
@@ -10,7 +10,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
   "Severity": "medium",
-  "ResourceType": "AwsEFSFileSystem",
+  "ResourceType": "AwsEfsAccessPoint",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/efs/efs_not_publicly_accessible/efs_not_publicly_accessible.metadata.json]:
```suggestion
  "ResourceType": "AwsEfsFileSystem",
```

File: prowler/providers/aws/services/emr/emr_cluster_account_public_block_enabled/emr_cluster_account_public_block_enabled.metadata.json [hunk 32]
```diff
@@ -7,7 +7,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:aws:emr:region:account-id",
   "Severity": "high",
-  "ResourceType": "AwsEMR",
+  "ResourceType": "Other",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/emr/emr_cluster_account_public_block_enabled/emr_cluster_account_public_block_enabled.metadata.json]:
```suggestion
  "ResourceType": "AwsAccount",
```

File: prowler/providers/aws/services/eventbridge/eventbridge_bus_exposed/eventbridge_bus_exposed.metadata.json [hunk 33]
```diff
@@ -7,7 +7,7 @@
   "SubServiceName": "",
   "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
   "Severity": "high",
-  "ResourceType": "AwsEventsEventbus",
+  "ResourceType": "AwsEventsEndpointEventBuses",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/eventbridge/eventbridge_bus_exposed/eventbridge_bus_exposed.metadata.json]:
```suggestion
  "ResourceType": "AwsEventsEventbus",
```

**Review by MrCloudSec — APPROVED** [review]

**Pull request merged**